pkg/profile/artifact.go

```diff
@@ -140,52 +133,90 @@ func (p *Profile) getResourceIfExists(ctx context.Context, res client.Object) (b
 // MakeArtifacts creates and returns a slice of runtime.Object values, which if
 // applied to a cluster would deploy the profile as a HelmRelease.
 func (p *Profile) MakeArtifacts() ([]runtime.Object, error) {
```

Comment:
is this func redundant now? We can just have the private one public

Reply:
@aclevername I don't know what else might call this. Is there anything outside profile which might call this in any ways? I didn't dare deleting it.